containers/package-builder/checkout-concent-version.sh.j2

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+DIRECTORY=$1
+
+# Check if a directory exists
+if [ -d "$DIRECTORY" ]; then
+    git fetch
+else
+    git clone /var/concent-data/concent/ "$DIRECTORY"
+    cd "$DIRECTORY"
+    git fetch
+    git checkout {{ concent_version }}
```

Comment:
Please make it a parameter of the script.

Reply:
Could you be more specific ? How do you return the template variable at the script input?

Reply:
I mean that instead of this you could have something like
``` bash
version=$2
git checkout $version
```
and call the script like
``` bash
./checkout-concent-version.sh <version>
```

Oh, but I can see the problem now. Version is embedded in `version.yml` and you don't have it directly in the makefile. Well, in that case let's leave it as is.

Reply:
Ok.